src/site/stages/build/process-cms-exports/transformers/node-event_listing.js

```diff
@@ -0,0 +1,56 @@
+const { getDrupalValue, utcToEpochTime } = require('./helpers');
+
+// Same as the createMetaTag in the transformer helpers, but uses
+// __typename instead of type. Because consistency.
+function createMetaTag(type, key, value) {
+  return {
+    __typename: type,
+    key,
+    value,
+  };
+}
+
+const transform = entity => {
+  const metaTags = entity.metatag.value;
+
+  return {
+    entityType: 'node',
+    entityBundle: 'event_listing',
+    entityUrl: {
+      // TODO: Get the breadcrumb from the CMS export when it's available
+      breadcrumb: [],
+      path: entity.path[0].alias,
+    },
+    entityMetatags: [
+      createMetaTag('MetaValue', 'title', metaTags.title),
+      createMetaTag('MetaValue', 'twitter:card', metaTags.twitter_cards_type),
+      createMetaTag('MetaProperty', 'og:site_name', metaTags.og_site_name),
+      createMetaTag(
+        'MetaValue',
+        'twitter:description',
+        metaTags.twitter_cards_description,
+      ),
+      createMetaTag('MetaValue', 'description', metaTags.description),
+      createMetaTag('MetaValue', 'twitter:title', metaTags.twitter_cards_title),
+      createMetaTag('MetaValue', 'twitter:site', metaTags.twitter_cards_site),
+      createMetaTag('MetaProperty', 'og:title', metaTags.og_title),
+      createMetaTag('MetaProperty', 'og:description', metaTags.og_description),
+    ],
```

Comment:
There is a [helper](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/process-cms-exports/transformers/helpers.js#L117-L149) for this that can be used like:

```js
entityMetatags: createMetatagArray(entity.metatag.value)
```

Reply:
updated